spec/lighthouse/audit_service_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'lighthouse/audit_service'
+
+RSpec.describe AuditService do
+  let(:runner) { double }
```

Comment:
🐘 this is not aligned (vs the rest of your tests) 😜

Reply:
Ugh, frustrating to have missed this! Will fix.